packages/ai-google/src/node/google-language-model.ts

```diff
@@ -32,27 +31,44 @@ import { CancellationToken } from '@theia/core';
 import { GoogleGenAI, FunctionCallingConfigMode, FunctionDeclaration, Content, Schema, Part, Modality, FunctionResponse, ToolConfig } from '@google/genai';
 import { wait } from '@theia/core/lib/common/promise-util';
 import { GoogleLanguageModelRetrySettings } from './google-language-models-manager-impl';
+import { UUID } from '@theia/core/shared/@lumino/coreutils';
```

Comment:
I meant `generateUuid` from `@theia/core` ;)

Reply:
Ah I see, thanks! I take a note that for the follow up then :)